roles/cluster_infra/tasks/main.yml

```diff
@@ -54,7 +59,7 @@
 
 - name: Template Terraform files into project directory
   template:
-    src: "{{ item }}.j2"
+    src: "{{ cluster_terraform_template_dir ~ '/' if cluster_terraform_template_dir is defined else '' }}{{ item }}.j2"
```

Comment:
This is a very long line. I think maybe something like this might be better?

```
- name: Template Terraform files into project directory
  template:
    src: >-
      {{-
        "{}{}.j2".format(
          (
            cluster_terraform_template_dir ~ "/"
            if cluster_terraform_template_dir is defined
            else ""
          ),
          item
        )
      }}
```

Reply:
Sure, updated.

Reply:
Tested and working